diff -p -up cooledit-3.17.17/widget/Makefile.am.undlnk cooledit-3.17.17/widget/Makefile.am
--- cooledit-3.17.17/widget/Makefile.am.undlnk	2009-12-04 23:16:48.000000000 +0100
+++ cooledit-3.17.17/widget/Makefile.am	2009-12-04 23:17:23.000000000 +0100
@@ -21,5 +21,7 @@ libCw_la_SOURCES = initapp.c coolwidget.
 
 libCw_la_LDFLAGS = -version-info 1:0:0
 
+libCw_la_LIBADD = -lX11
+
 DEFS = @X_CFLAGS@ -DLOCALEDIR=\"$(localedir)\" -DLIBDIR=\"${exec_prefix}/share/cooledit\" @DEFS@ 
 
diff -p -up cooledit-3.17.17/widget/Makefile.in.undlnk cooledit-3.17.17/widget/Makefile.in
--- cooledit-3.17.17/widget/Makefile.in.undlnk	2009-12-04 23:17:00.000000000 +0100
+++ cooledit-3.17.17/widget/Makefile.in	2009-12-04 23:17:40.000000000 +0100
@@ -57,7 +57,7 @@ am__strip_dir = `echo $$p | sed -e 's|^.
 am__installdirs = "$(DESTDIR)$(libdir)"
 libLTLIBRARIES_INSTALL = $(INSTALL)
 LTLIBRARIES = $(lib_LTLIBRARIES)
-libCw_la_LIBADD =
+libCw_la_LIBADD = -lX11
 am_libCw_la_OBJECTS = initapp.lo coolwidget.lo dirtools.lo menu.lo \
 	mathfuncs.lo filebrowser.lo coolnext.lo bitmapbutton.lo \
 	pixmap.lo dialog.lo disable.lo focus.lo xdnd.lo ClientWin.lo \
